spring-boot-admin-samples/spring-boot-admin-sample-servlet/src/main/java/de/codecentric/boot/admin/SecuritySecureConfig.java

```diff
@@ -50,6 +50,7 @@ protected void configure(HttpSecurity http) throws Exception {
 
 		http.authorizeRequests(
 				(authorizeRequests) -> authorizeRequests.antMatchers(this.adminServer.path("/assets/**")).permitAll() // <1>
+						.antMatchers(this.adminServer.path("/actuator/**")).permitAll()
```

Comment:
Only the requests to /info and /health should be unprotected (reflecting the boot default).

Reply:
I replaced by 2 different entries, to make it more clear and readable (since this is a sample project).
Not sure if a Regex (or any other more condensed expression) would be preferred